src/frontend/src/styles/main.css

```diff
@@ -203,6 +204,9 @@
 
   --rc-button--disabled: var(--vc-shadow);
 
+  --rc-landing-page-background: var(--vc-brand-gradient);
```

Comment:
--rc is for color, not a gradient. it would be rg or something I guess.

But I think  `--rc-landing-page-background` is too specific anyway. It allows for almost no reuse. Maybe calling it something like --rg-highlight? I would discuss with Artem for what reason the gradients are used and use this as the name,

Reply:
I prefer to keep it as `rc`. A color could easily replace the gradient in the future and I don't want to change the name of the variable then.